native/src/hooks/useTtsPlayer.ts

```diff
@@ -13,16 +14,18 @@ const useTtsPlayer = (model?: PageModel | LocalNewsModel | TunewsModel | undefin
   const [previousSentences] = useState(ttsContext.sentences)
   const { setSentences } = ttsContext
 
-  useLayoutEffect(() => {
-    if (model && model.content.length > 0) {
-      const content = parseHTML(model.content)
-      const sentences: string[] = segment(languageCode, content)
-      setSentences([model.title, ...sentences].filter(sentence => sentence.length > 0))
-    } else {
-      setSentences([])
-    }
-    return () => setSentences(previousSentences)
-  }, [previousSentences, setSentences, model, languageCode])
+  useFocusEffect(
+    useCallback(() => {
+      if (model && model.content.length > 0) {
+        const content = parseHTML(model.content)
+        const sentences: string[] = segment(languageCode, content)
+        setSentences([model.title, ...sentences].filter(sentence => sentence.length > 0))
+      } else {
+        setSentences([])
+      }
+      return () => setSentences(previousSentences)
+    }, [previousSentences, setSentences, model, languageCode]),
+  )
```

Comment:
Only wrapped this in useFocusEffect/useCallback

Reply:
Nice